patient-db/src/main/java/com/hartwig/hmftools/patientdb/clinical/lims/cohort/LimsCohortModel.java

```diff
@@ -20,67 +20,15 @@ public abstract class LimsCohortModel {
 
     @Nullable
     public LimsCohortConfig queryCohortData(@Nullable String cohortString, @NotNull String sampleId) {
-        String effectiveCohortString = cohortString;
         if (cohortString == null || cohortString.isEmpty()) {
-            effectiveCohortString = resolveFromSampleId(sampleId);
-            if (effectiveCohortString != null) {
-                LOGGER.warn("No cohort string present in LIMS for sample '{}'. Cohort has been set to {}", sampleId, effectiveCohortString);
-            }
-        }
-
-        LimsCohortConfig cohortConfigData = limsCohortMap().get(effectiveCohortString);
-        if (cohortConfigData == null) {
-            LOGGER.warn("Could not resolve cohort config for sample '{}' based on LIMS cohort '{}'", sampleId, cohortString);
+            LOGGER.warn("No cohort string present in LIMS for sample '{}'.", sampleId);
             return null;
-        } else {
-            if (cohortConfigData.cohortId().contains("CPCT") || cohortConfigData.cohortId().contains("DRUP")) {
-                if (sampleId.startsWith(cohortConfigData.cohortId().substring(0, 4))) {
-                    return cohortConfigData;
-                } else {
-                    throw new IllegalStateException("Sample '" + sampleId + "' does not seem to be part of CPCT/DRUP cohort");
-                }
-            } else {
-                return cohortConfigData;
+        }else{
```

Comment:
formatting. This else can also be removed because the code is returned in the if statement above

Reply:
Removed the else, not needed at all indeed.